frontend/src/screens/Profile/index.tsx

```diff
@@ -1,5 +1,175 @@
+import { Form, FormControl, FormDescription, FormField, FormItem, FormMessage } from '~/components/Form'
+import { Button } from '../../components/Button'
+import { Input } from '~/components/Input'
+import { useForm } from 'react-hook-form'
+import cameraIcon from '~/assets/icons/camera.icon.svg'
+import { ChangeEvent, useState } from 'react'
+import { cva } from 'class-variance-authority'
+import { cn } from '~/__generated__/utils'
+import { type IUser } from '~/types/user.type'
+
+const tabVariant = cva('h-1 w-full', {
+  variants: {
+    variant: {
+      default: 'bg-transparent',
+      active: 'bg-orange-primary',
+    },
+  },
+  defaultVariants: {
+    variant: 'default',
+  },
+})
+
 const Profile = () => {
-  return <div>Profile</div>
+  const img = ''
+  const [toggleState, setToggleState] = useState(1)
+  const [imageUrl, setImageUrl] = useState('')
+  const form = useForm()
+
+  const toggleTab = (index: number) => {
+    setToggleState(index)
+  }
+
+  const handleImageChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const file = event.target.files?.[0]
+    const reader = new FileReader()
+
+    reader.onload = (e) => {
+      if (e.target) {
+        setImageUrl(e.target.result as string)
+      }
+    }
+
+    if (file) {
+      reader.readAsDataURL(file)
+    }
+  }
+  return (
+    <div className="flex h-screen w-full items-center justify-center bg-[--gray-secondary]">
+      <Form {...form}>
+        <form className="absolute flex h-full w-full flex-col overflow-hidden rounded-lg bg-white p-6 sm:top-[10%] sm:max-h-[855px] sm:max-w-[650px]">
```

Comment:
Vì sao ở đây lại hidden để ko scoll xuống được vậy ?
```suggestion
        <form className="absolute flex h-full w-full flex-col rounded-lg bg-white p-6 sm:top-[10%] sm:max-h-[855px] sm:max-w-[650px]">
```